src/current/_includes/releases/v23.1/v23.1.15.md

```diff
@@ -0,0 +1,67 @@
+- Fixed a bug that prevented database [restore](https://www.cockroachlabs.com/docs/v23.1/restore) when the database contained a view or routine that referenced a user-defined type in the body string. For views, this bug was introduced in v20.2 when UDTs were introduced. For routines, this bug was introduced in v22.2 when UDFs were introduced. [#116903][#116903]
+- Fixed a durability issue in the Raft log storage mechanism due to improper synchronization of filesystem metadata, potentially leading to the loss of specific write operations (`AddSSTable`), notably utilized by operations such as `RESTORE`. This vulnerability was exposed only under conditions of power failure or operating system crashes, potentially causing CockroachDB to enter a crash loop upon restart. In extreme scenarios, such as a simultaneous power outage or crash across multiple nodes, it could result in an irrecoverable quorum loss. [#117383][#117383]
+- Fixed an issue in Raft log truncation that had the potential to cause crash loops and irretrievable quorum loss, especially in the rare but severe scenario where all replicas concurrently enter a crash loop. This issue emerged under conditions where the cluster was processing a bulk write operation (such as schema changes, imports, or restores), while a log truncation command was active, and the CockroachDB process experienced a crash. [#117299][#117299]
+- Fixed the total runtime value referenced in SQL stats, resolving the bug where the [Console](https://www.cockroachlabs.com/docs/v23.1/ui-overview) erroneously displayed percentages over 100%. [#117496][#117496]
+- The values for the current and past hour in the top Activity table are now calculated correctly, fixing the missing data issue in SQL stats and, consequently, on the [SQL Activity](https://www.cockroachlabs.com/docs/v23.1/ui-overview#sql-activity) page. [#118427][#118427]
+- Fixed a bug where CockroachDB would erroneously return an error if an empty search path parameter was encountered during search path setting. [#117556][#117556]
+- Fixed a bug in the [Row-Level TTL](https://www.cockroachlabs.com/docs/v23.1/row-level-ttl) job that would cause it to skip expired rows if the primary key of the table included columns of the collated string type. This bug was present since the initial release of row-level TTL in v22.2.0. [#117513][#117513]
+- Fixed a bug that could cause [`DELETE`](https://www.cockroachlabs.com/docs/v23.1/delete) queries sent by the [Row-Level TTL](https://www.cockroachlabs.com/docs/v23.1/row-level-ttl) job to use a secondary index rather than the primary index to find the rows to delete. This could lead to some `DELETE` operations taking a much longer time than they should. This bug was present since v22.2.0. [#118336][#118336]
+- Internal queries issued by the TTL jobs now use optimal plans. The bug has been present since v22.2. [#118564][#118564]
```

Comment:
Should this be included since it's internal? And is this a feature or a bug fix?